recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+make check > /dev/null
```

Comment:
I see. Is it so long it is killing the CIs? This is nice info to have if it goes wrong. Otherwise this failure has no context.